packages/bot-web-ui/src/components/run-panel.jsx

```diff
@@ -92,7 +92,7 @@ const drawerFooter = ({
                      by our system. Please be aware that some completed transactions may not be displayed
                      in the transaction table if the bot is stopped while placing trades. You may refer to
                      the statement page for details of all completed transactions.`)}
-                    zIndex='5'
+                    zIndex={5}
```

Comment:
Should be passed in as a constant from constants file.

Reply:
@aaron-binary we will update it later, cause its been a regression test i don't wanna add more stuff here.